doc/overview.rst

```diff
@@ -629,108 +639,81 @@ Tabular
 
+Using the model ID, define your model as a JumpStart model. Use the ``deploy`` method to automatically deploy your model for inference.
+In this example, we use the FLAN-T5 XL model from HuggingFace.
 .. code:: python
 
-   model_id, model_version = "huggingface-spc-bert-base-cased", "1.0.0"
-   scope = "training" # or "inference"
+    from sagemaker.jumpstart.model import JumpStartModel
 
-Then use those values to retrieve the model as follows.
+    model_id = "huggingface-text2text-flan-t5-xl"
+    my_model = JumpStartModel(model_id=model_id)
+    predictor = my_model.deploy()
+
+You can then run inference with the deployed model using the ``predict`` method.
 
 .. code:: python
 
-   from sagemaker import model_uris
+    question = "What is Southern California often abbreviated as?"
+    response = predictor.predict(question)
+    print(response)
 
-   model_uri = model_uris.retrieve(
-       model_id=model_id, model_version=model_version, model_scope=scope
-   )
+For more information about the ``JumpStartModel`` class and its parameters,
+see `JumpStartModel <https://sagemaker.readthedocs.io/en/stable/api/inference/model.html#sagemaker.jumpstart.model.JumpStartModel>`__.
 
-Model scripts
--------------
+Additional low-code deployment utilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To adapt pre-trained models for SageMaker, a custom script is needed to perform training
-or inference. SageMaker maintains a suite of scripts used for each of the models in the
-S3 bucket, which can be accessed using the SageMaker Python SDK Use the ``model_id`` and
-``version`` of the corresponding model to retrieve the related script as follows.
+You can optionally include specific model versions or instance types when deploying a pretrained model
+using the ``JumpStartModel`` class. All JumpStart models have a default instance type.
+Retrieve the default deployment instance type using the following code:
 
 .. code:: python
 
-   from sagemaker import script_uris
+    from sagemaker import instance_types
 
-   script_uri = script_uris.retrieve(
-       model_id=model_id, model_version=model_version, script_scope=scope
-   )
+    instance_type = instance_types.retrieve_default(
+        model_id=model_id,
+        model_version=model_version,
+        scope="inference")
+    print(instance_type)
 
-Model images
--------------
+See all supported instance types for a given JumpStart model with the ``instance_types.retrieve()`` method.
 
-A Docker image is required to perform training or inference on all
-SageMaker models. SageMaker relies on Docker images from the
-following repos https://github.com/aws/deep-learning-containers,
-https://github.com/aws/sagemaker-xgboost-container,
-and https://github.com/aws/sagemaker-scikit-learn-container. Use
-the ``model_id`` and ``version`` of the corresponding model to
-retrieve the related image as follows.
+To check valid data input and output formats for inference, you can use the ``retrieve_options()`` method
```

Comment:
can we also mention the content type and accept type utilities?
- https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/content_types.py
- https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/accept_types.py